src/main/logging/README.md

```diff
@@ -0,0 +1,244 @@
+## Deploy Logging Pattern - EFK Stack
+
+A common Kubernetes logging pattern is a combination of Fluent, Elasticsearch, and Kibana which is known as *EFK Stack*. 
+
+`Fluent` is an open source and multi-platform log processor which collects data/log from different sources, aggregates and forwards them to multiple destinations and is fully compatible with Docker and Kubernetes environments. 
+
+`Elasticsearch` is a distributed open search and analytics engine for all types of data. 
+
+`Kibana` is an open source frontend application that sits on top of Elasticsearch, providing search and data visualization capabilities for data indexed in Elasticsearch.
+
+There are different methods to deploy EFK stack and here we provide two deployment methods, first deploy EFK using helm chart and deploy local elasticsearch on Kubernetes and second using managed elasticsearch (EKS and AWS ES).
+
+## EFK using local elasticsearch
+
+This solution deploys all EFK stack inside the k8s cluster locally. By enabling fluentd, helm_install script installs fluentd in EKS. It means after running helm_install all product pods will run fluentd which collects all log files and sends them to the fluentd aggregator container. 
```

Comment:
I was looking through the charts and I cannot see any centralised logging fluentd pod that would collect the logs. I think every fluentd sidecar is sending the logs directly to elasticsearch.

Reply:
I agree, I moved the order